explainaboard/utils/eval_utils/qa_table_text_hybrid.py

```diff
@@ -0,0 +1,328 @@
+"""Pre-processing functions for qa-table_text_hybrid task
+The pre-processing is so complicated that I put most of the original processing code
+here. The official evaluation script can be found here:
+https://github.com/NExTplusplus/L2I/blob/main/evaluate.py
+"""
+
+from __future__ import annotations
+
+import re
+import string
+from typing import Union
+
+import numpy as np
+from scipy.optimize import linear_sum_assignment
+
+
+def negative_num_handle(x: str):
```

Comment:
The name `*_handle` looks not informative. How about `*_factor` or `*_coefficient`?